### PR TITLE
Dashboard refactoring and added ongoing trip functionality

### PR DESCRIPTION
**Pull request by morgankbender**

### CODE REVIEW

File: src/components/dashboard/jumbotron/OngoingJumbotron.js [hunk 1]
```diff
@@ -39,8 +42,8 @@ export default class CurrentJumbotron extends Component {
 
       return (
         <div>
-          <p>{upcoming.name}</p>
           <p>{upcoming.dscript}</p>
+          <p />
```
Comment (gzinck) [inline src/components/dashboard/jumbotron/OngoingJumbotron.js]: Is this necessary? Might be for styling...

File: src/components/dashboard/jumbotron/OngoingJumbotron.js [hunk 2]
```diff
@@ -81,8 +84,28 @@ export default class CurrentJumbotron extends Component {
...
     return (
       <div>
-        {this.getCurrentDiv()}
-        {this.getUpcomingDiv()}
+        <p>You have an ongoing trip!</p>
+        <h2>{this.props.trip.name}</h2>
+        {this.getCurrentEventsDiv()}
+        {this.getUpcomingEventsDiv()}
+
+        <div className="flex-wrap-center">
```
Comment (gzinck) [inline src/components/dashboard/jumbotron/OngoingJumbotron.js]: Niiice

File: src/components/dashboard/jumbotron/OngoingJumbotronContainer.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import OngoingJumbotron from './OngoingJumbotron';
+import { getSortedTrips } from '../../../redux/getters/getTrips';
+
+const mapStateToProps = (state, ownProps) => {
+  let activeEvents = getActiveEvents(state, ownProps.trip_id);
+
+  return {
+    currEvents: activeEvents.current,
+    upcomingEvents: activeEvents.upcoming,
+    trip: getSortedTrips(state).active.shift()
```
Comment (gzinck) [inline src/components/dashboard/jumbotron/OngoingJumbotronContainer.js]: What is this one? Is this getting the first upcoming event?

**Review by gzinck — COMMENTED** [review]
Looking pretty good :)

File: src/components/dashboard/jumbotron/OngoingJumbotronContainer.js (hunk 3 above)
Comment (morgankbender) [reply src/components/dashboard/jumbotron/OngoingJumbotronContainer.js]: The current trip. I'm going to make this a little nicer though :)

File: src/components/dashboard/jumbotron/OngoingJumbotron.js (hunk 1 above)
Comment (morgankbender) [reply src/components/dashboard/jumbotron/OngoingJumbotron.js]: Nope. Accidental. Good-bye weird <p />

**Review by evscott — APPROVED** [review]
This looks great.

**Pull request merged**